File: C++/silver/2563.cpp
#include <iostream>
#include <vector>

using namespace std;

int main(){
    int T, x_value, y_value, result = 0, x_max = -1, y_max = -1;
    cin >> T;
    vector<vector<int>> paper(100, vector<int>(100, 0));
    for(int i = 0; i < T; i++){
        cin >> x_value >> y_value;
        if(x_value > x_max){
            x_max = x_value;
        }
        if(y_value > y_max){
            y_max = y_value;
        }
        for(int j = x_value; j < x_value + 10; j++){
            for(int k = y_value; k < y_value + 10; k++){
                paper[j][k] = 1;
            }
        }
    }
    for(int i = 0; i < x_max+10; i++){
        for(int j = 0; j< y_max+10; j++){
            if(paper[i][j] == 1) result++;
        }
    }
    cout << result;
    return 0;
}

File: python/silver/2563.py
paper = [[0] * 100 for _ in range(100)]
x_max, y_max = -1, -1
result = 0
T = int(input())
for _ in range(T):
    x_value, y_value = map(int, input().split())
    x_max = max(x_value, x_max)
    y_max = max(y_value, y_max)
    for i in range(x_value, x_value+10):
        for j in range(y_value, y_value+10):
            paper[i][j] = 1

for i in range(x_max+10):
    result += paper[i].count(1)

print(result)
